mempool/reactor.go

```diff
@@ -30,14 +31,16 @@ type Reactor struct {
 	// connections for different groups of peers.
 	activePersistentPeersSemaphore    *semaphore.Weighted
 	activeNonPersistentPeersSemaphore *semaphore.Weighted
+	traceClient                       trace.Tracer
 }
 
 // NewReactor returns a new Reactor with the given config and mempool.
-func NewReactor(config *cfg.MempoolConfig, mempool *CListMempool) *Reactor {
+func NewReactor(config *cfg.MempoolConfig, mempool *CListMempool, traceClient trace.Tracer) *Reactor {
```

Comment:
[not blocking]

I guess it doesn't hurt to add the tracer here, but we don't have any plans to trace this mempool, and we aren't adding any traces here afaict so we could also remove it